packages/examples/demo-app/.cache_12camz2

```diff
@@ -0,0 +1 @@
+[{"/Users/dlabaj/repos/appservices-patternfly/packages/examples/demo-app/src/App.tsx":"1","/Users/dlabaj/repos/appservices-patternfly/packages/examples/demo-app/src/index.tsx":"2","/Users/dlabaj/repos/appservices-patternfly/packages/examples/demo-app/src/react-app-env.d.ts":"3"},{"size":8193,"mtime":1596113709419},{"size":268,"mtime":1596072133403,"results":"4","hashOfConfig":"5"},{"size":40,"mtime":1596076108433,"results":"6","hashOfConfig":"5"},{"filePath":"7","messages":"8","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"uxu0ez",{"filePath":"9","messages":"10","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/dlabaj/repos/appservices-patternfly/packages/examples/demo-app/src/index.tsx",[],"/Users/dlabaj/repos/appservices-patternfly/packages/examples/demo-app/src/react-app-env.d.ts",[]]
```

Comment:
looks like a third temp cache file, probably not intended to merge in with your other changes